Source/Core/Core/IOS/USB/Emulated/Skylander.h

```diff
@@ -43,7 +43,7 @@ enum class Element
 
 struct SkyData
 {
-  const char* name = "";
+  const char* name;
```

Comment:
Any reason you removed this default...?

Reply:
None of the list items use the default name, so removed it.

Reply:
Have restored now